### PR TITLE
Java 11 readiness: build both on JDK8 and JDK11

### PR DESCRIPTION
**Pull request by batmat**
Making sure this plugin is constantly checked both building with JDK8 and JDK11.

This goes with the Jenkins Java 11 General Availability announcement made back in March, and we're making a pass to check plugins are looking good on a JDK11.

([to understand the Jenkinsfile content](https://wiki.jenkins.io/display/JENKINS/Java+11+Developer+Guidelines#Java11DeveloperGuidelines-MakesureyourpluginistestedinContinuousIntegrationonJava8andJava11atthesametime))

@jenkinsci/java11-support

### CODE REVIEW

**Comment by alecharp** [comment]
I added the `dependencyManagement` but it is not required anymore, with the correct version of plugins. Sorry I added a commit in this PR for nothing..

**Comment by alecharp** [comment]
I think this is ready for review @jenkinsci/java11-support

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.32</version>
+    <version>3.43</version>
```
Comment (oleg-nenashev) [inline pom.xml]:
```suggestion
    <version>3.47</version>
```

**Comment by batmat** [comment]
:heavy_check_mark:

**Pull request merged**